<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindKeysDialog</class>
 <widget class="QDialog" name="FindKeysDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Поиск ключей</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButtonDatabase">
        <property name="text">
         <string>Сервер ключей</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxFirms">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentText">
         <string>Выберите организацию</string>
        </property>
        <property name="maxVisibleItems">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButtonFolder">
        <property name="text">
         <string>Локальная папка</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>№ ЭККА:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditZN"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButtonFind">
        <property name="text">
         <string>Найти</string>
        </property>
        <property name="icon">
         <iconset resource="../recource.qrc">
          <normaloff>:/Icons/find.png</normaloff>:/Icons/find.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lineEditPath"/>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonFolder">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../recource.qrc">
            <normaloff>:/Icons/folder.png</normaloff>:/Icons/folder.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../recource.qrc"/>
 </resources>
 <connections/>
</ui>
